tests/committable.py

```diff
@@ -0,0 +1,63 @@
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the Apache 2.0 License.
+import infra.e2e_args
+import infra.network
+import infra.proc
+import time
+
+from loguru import logger as LOG
+
+
+def run(args):
+    hosts = ["localhost"] * 5
+
+    with infra.network.network(
+        hosts, args.binary_dir, args.debug_nodes, args.perf_nodes, pdb=args.pdb
+    ) as network:
+        network.start_and_join(args)
+        primary, backups = network.find_nodes()
+
+        # Suspend three of the backups to prevent commit
+        backups[1].suspend()
+        backups[2].suspend()
+        backups[3].stop()
```

Comment:
What's the difference between `suspend()` and `stop()` here? (We suspend 2 and stop 1?)

Reply:
Suspend allows resuming, stop is killing the node. We do not need 3 anymore, since we want only three nodes to force a unanimous election, and it's unnecessary to operate after that. We only ever start backup 4 because we want f=2.